### ISSUE TITLE
CSS class issue

### ISSUE THREAD

**Issue by fridobox** [bywatersolutions/koha-plugin-coverflow#38]
I see CSS code in opac_head with selector ".coverflow" :

https://github.com/bywatersolutions/koha-plugin-coverflow/blob/4a1819c046b45a52d5de7df6c626206ed4320235/Koha/Plugin/Com/ByWaterSolutions/CoverFlow.pm#L322

But the CoverFlow has class "koha_coverflow".
Looks like this CSS is actually not active.

Also the selector ".itemTitle" should be ".coverflow .itemTitle" no ?
https://github.com/bywatersolutions/koha-plugin-coverflow/blob/4a1819c046b45a52d5de7df6c626206ed4320235/Koha/Plugin/Com/ByWaterSolutions/CoverFlow.pm#L340

But maybe this code is obsolete ?

**Comment by ricofreak** [comment]
Yes, still valid. But I think we should removed the .coverflow CSS that is now doing nothing. If we add it to .koha-coverflow it will break a lot of existing implementations.
